embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/injection/InitModule.kt

```diff
@@ -48,6 +48,11 @@ internal interface InitModule {
      */
     val tracer: Tracer
 
+    /**
+     * Composite exporter that can be used to add multiple SpanExporters at runtime
+     */
+    val compositeSpanExporter: CompositeSpanExporter
```

Comment:
Instead of directly exposing this, what do you think about persisting this a `List` somewhere so we can add them later? That way, we won't have to expose the composite exporter outside of `OpenTelemetrySdk`, making it an implementation detail that we can change later, i.e. change to use the default composite once we get this initializing later.